src/node/constant_source.rs

```diff
@@ -11,8 +11,11 @@ use super::{AudioNode, AudioScheduledSourceNode, ChannelConfig};
 // dictionary ConstantSourceOptions {
 //   float offset = 1;
 // };
+// @note: this does not extend AudioNodeOptions in the spec, why?
+// https://webaudio.github.io/web-audio-api/#ConstantSourceOptions
```

Comment:
`AudioNodeOptions` are useless for source nodes, because they instruct how to upmix the inputs.
This is a common source of confusion, see e.g. https://github.com/mdn/content/pull/18472
Also for the spec editors because they accidentally let OscillatorOptions extend AudioNodeOptions

Reply:
Added your note and link to the comment, so we don't forget again. I will try to align this to other nodes as well later